mm2src/mm2_p2p/src/application/request_response/mod.rs

```diff
@@ -12,10 +12,6 @@ use serde::{Deserialize, Serialize};
 pub enum P2PRequest {
     /// Request for order matching.
     Ordermatch(ordermatch::OrdermatchRequest),
-    /// Request for network information from the target peer.
-    ///
-    /// TODO: This should be called `PeerInfoRequest` instead. However, renaming it
-    /// will introduce a breaking change in the network and is not worth it. Do this
-    /// renaming when there is already a breaking change in the release.
-    NetworkInfo(network_info::NetworkInfoRequest),
```

Comment:
This PR already leads to breaking change. It's perfect time to rename this.

Reply:
One way to avoid breaking changes is to have a timeout on the request for peer timestamp and allow them to connect if there is no response from the peer. We later remove this when most peers update. Another way is to get the mm2 version and do the timestamp request depending on the version. What do you think?

Reply:
> One way to avoid breaking changes is to have a timeout on the request for peer timestamp and allow them to connect if there is no response from the peer. We later remove this when most peers update.

This sounds like a good plan!

> Another way is to get the mm2 version and do the timestamp request depending on the version. What do you think?

I think the first idea was better, but then I need to revert the peer-info renaming part. What about bundling these changes in some branch, and then releasing multiple breaking changes at once (like removing mm2 binaries and some other stuff) ?